styles.css

```diff
@@ -0,0 +1,50 @@
+/* Add a black background color to the top navigation */
+.topnav {
+    background-color: #333;
+    overflow: hidden;
+    viewport-fit: top;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
+    background-color: #04AA6D;
+    color: white;
+}
+
+.project-tile {
+    text-decoration: center;
+}
+
+header {
+    position: fixed;
+    top: 0px;
```

Comment:
I kindly suggest you change this value for this header top property since it is not a valid value. One thing you can try to use linters as a helper to check your CSS styling so that you can have a clearer view of CSS🤗